### PR TITLE
GKE deployer: Port a few more projects

### PR DESCRIPTION
**Pull request by zmerlynn**
Along the way: Add gke-staging, gke-prod and ginkgo-parallel "platforms". These are *very* temporary, and intended to be replaced by kubetest flags soon, next kubetest iteration. They are easy things to replace while I port a few more jobs.

### CODE REVIEW

File: jenkins/bootstrap_test.py [hunk 1]
```diff
@@ -2296,6 +2296,8 @@ def test_valid_job_envs(self):
                     if (
                             '--env-file=jobs/platform/gce.env' not in args
                             and '--env-file=jobs/platform/gke.env' not in args
+                            and '--env-file=jobs/platform/gke-staging.env' not in args
+                            and '--env-file=jobs/platform/gke-prod.env' not in args
```
Comment (krzyzacy) [inline jenkins/bootstrap_test.py]: gke jobs still references to `jobs/platform/gke.env` so those two might not needed here.

**Comment by krzyzacy** [comment]
LGTM, please fix the lint

File: jenkins/bootstrap_test.py (hunk 1 above)
Comment (zmerlynn) [inline jenkins/bootstrap_test.py]: I flipped a couple to only reference one of them (now that `gke.env` is just an endpoint selection).

Comment (krzyzacy) [inline jenkins/bootstrap_test.py]: ah I see :-) didn't notice that's the only thing left in gke.env.

**Comment by zmerlynn** [comment]
Done, thanks!

**Comment by foxish** [comment]
These appear to have broken the staging jobs:

* https://k8s-testgrid.appspot.com/google-gke-staging#gci-gke-staging&show-stale-tests=
* https://k8s-testgrid.appspot.com/google-gke-staging#gci-gke-staging-parallel&show-stale-tests=

**Pull request merged**